src/gui/SetupDialog.cpp

```diff
@@ -137,12 +139,14 @@ SetupDialog::SetupDialog( ConfigTabs _tab_to_open ) :
 	m_displayWaveform(ConfigManager::inst()->value( "ui",
 						   "displaywaveform").toInt() ),
 	m_disableAutoQuit(ConfigManager::inst()->value( "ui",
-						   "disableautoquit").toInt() )
+						   "disableautoquit").toInt() ),
+	m_vstEmbedMethod(ConfigManager::inst()->value( "ui",
+						   "vstembedmethod", "xembed"))
```

Comment:
Is the default value `"xembed"` fine for non-linux platforms?

Reply:
If I understand correctly, `ConfigManager::inst()->vstEmbedMethod()` could be used instead.

Reply:
`ConfigManager::inst()->vstEmbedMethod()` is better here indeed, thanks.